### PR TITLE
Logo construct

### PR DESCRIPTION
**Pull request by dquach217**
- Add the Construct logo
- Move the Verizon Logo
- Fix the sizing
- Change the max-width to 1000px

### CODE REVIEW

**Comment by dquach217** [comment]
Fixes #708

File: app/assets/stylesheets/home/layout.sass [hunk 1]
```diff
@@ -218,7 +218,7 @@ body
 
 .sponsor__wrapper
   margin: 0 auto
-  max-width: 800px
+  max-width: 1000px
```
Comment (sman591) [inline app/assets/stylesheets/home/layout.sass]: Why this change?

Comment (sman591) [reply app/assets/stylesheets/home/layout.sass]: I see, just seems like they're already the biggest thing on the page

screen shot 2019-02-04 at 11 56 06 am

Comment (dquach217) [reply app/assets/stylesheets/home/layout.sass]: It's also because of the fact that we moved Verizon on the same level as Constellation, GCP, Wegmans
